### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section:
 - I added a new JavaScript file and imported it as a module:
 - I implemented status updates (completed: true / false).
 - I added an event listener to the checkbox (change).
 - I updated the item's object's value for the completed key upon user actions.
 - I stored the updated array of items in local storage, so the user gets the correct list values after the page reloads.
 - Below is the live link to the project:
    https://raw.githack.com/menyagah/TodoList-webpack/feature/dist/index.html
- Fixed Tse recommended issues

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,11 +19,23 @@ const todoList = [{
...
     <div class="delete-icon"></div>`;
   data.appendChild(div);
-});
\ No newline at end of file
+});
+
+for (let i = 0; i < todos.length; i += 1) {
+  const inCheck = document.querySelector(`#check-${todos[i].id}`);
+  inCheck.addEventListener('input', (e) => {
+    checkItem(i, e, todos);
```
Comment (EricMbouwe) [inline src/index.js]: - [ ] Although the `input` event handler is working fine you are required to use the `change` event. Those two behave differently, for example, the change event is not necessarily fired for each alteration to an element's value. You can learn more about [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event)

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @menyagah,

Good job so far, your code is really easy to read! Keep up doing a good job.
- [ ] Please make sure to have a meaningful PR title and a meaningful feature branch name.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] Use a better color for your task texts. Currently, they are not really legible.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
Thank you @EricMbouwe . Working on the Issues.

**Review by EricMbouwe — APPROVED** [review]
Hi @menyagah, 

# Status: Approved✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
Thank You!

**Pull request merged**